### PR TITLE
added flavor texts

### PR DESCRIPTION
**Pull request by aramshiva**
I've added some more flavors, some festive and some honey lavender!

### CODE REVIEW

**Review by sampoder — CHANGES_REQUESTED** [review]
looks like some awesome additions! might want to remove the git comments: "<<<<<<< Updated upstream" etc.

**Comment by aramshiva** [comment]
removed comments

File: app/services/flavor_text_service.rb [hunk 1]
```diff
@@ -87,6 +87,7 @@ def holiday_flavor_texts
       "Dear Santa...",
       "where's my gingerbread house",
       "makin' that money snow"
```
Comment (sampoder) [inline app/services/flavor_text_service.rb]:
```suggestion
      "makin' that money snow",
```

File: app/services/flavor_text_service.rb [hunk 2]
```diff
@@ -99,6 +100,7 @@ def spooky_flavor_texts
       "🧛",
       "🎃",
       "Pumpkin spice is the pumpkin spice of life."
```
Comment (sampoder) [inline app/services/flavor_text_service.rb]:
```suggestion
      "Pumpkin spice is the pumpkin spice of life.",
```

**Review by sampoder — CHANGES_REQUESTED** [review]
just a couple of commas to add!

**Comment by aramshiva** [comment]
sorry about that! fixed

**Review by sampoder — APPROVED** [review]

File: app/services/flavor_text_service.rb [hunk 3]
```diff
@@ -258,6 +260,7 @@ def flavor_texts
       "Open on weekdays!",
       "Open on #{Date.today.strftime("%A")}s",
       "??? profit!",
+      "Made with Rails, Ruby and did I forgot to mention Rails?",
```
Comment (garyhtou) [inline app/services/flavor_text_service.rb]: i'm not too sure how grammar works lol 🤷‍♂️ this suggestion could be wrong
```suggestion
      "Made with Rails, Ruby and did I forget to mention Rails?",
```

**Review by garyhtou — APPROVED** [review]
good to go! yay 🎃

**Pull request merged**